test/e2e/suite/mtls/mtls.go

```diff
@@ -54,6 +54,18 @@ var _ = framework.CasesDescribe("mTLS correctness", func() {
 			},
 		}
 
+		injectLabel = func() map[string]string {
+			if f.Config().AmbientEnabled {
+				return map[string]string{
+					"istio.io/dataplane-mode": "ambient",
+				}
+			} else {
+				return map[string]string{
+					"istio-injection": "enabled",
+				}
+			}
+		}()
```

Comment:
question: Maybe an obvious question (and not really related to the PR), but just checking I've understood right: I guess there's no situation where you'd set dataplane-mode to ambient and enable injection? I can't imagine why that would be desirable, just want to check.

Reply:
Thanks, that's really helpful 👍